WorldEdit/WorldEdit.cs

```diff
@@ -1135,6 +1431,97 @@ private void SetWall(CommandArgs e)
 			}
 		}
 
+		private void SetGrass(CommandArgs e)
+		{
+			if (e.Parameters.Count == 0)
+			{
+				e.Player.SendErrorMessage("Invalid syntax! Proper syntax: //setgrass <grass> [=> boolean expr...]");
+				return;
+			}
+			PlayerInfo info = e.Player.GetPlayerInfo();
+			if (info.X == -1 || info.Y == -1 || info.X2 == -1 || info.Y2 == -1)
+			{
+				e.Player.SendErrorMessage("Invalid selection!");
+				return;
+			}
+
+			object[] grassTiles = new object[]
```

Comment:
You should use `Biomes` to get grass tile ids.
This would be initialized in `OnInitialize()`.
Here are codes:
```csharp
			#region Biomes
			// Format: dirt, stone, ice, sand, grass, plants, tall plants, vines, thorn

			Biomes.Add("crimson", new[] { 0, 203, 200, 234, 199, -1, -1, 205, 32 });
			Biomes.Add("corruption", new[] { 0, 25, 163, 112, 23, 24, -1, -1, 32 });
			Biomes.Add("hallow", new[] { 0, 117, 164, 116, 109, 110, 113, 52, -1 });
			Biomes.Add("jungle", new[] { 59, 1, 161, 53, 60, 61, 74, 62, 69 });
			Biomes.Add("mushroom", new[] { 59, 1, 161, 53, 70, 71, -1, -1, -1 });
			Biomes.Add("normal", new[] { 0, 1, 161, 53, 2, 3, 73, 52, -1 });
			Biomes.Add("snow", new[] { 147, 161, 161, 53, 147, -1, -1, -1, -1 });
			#endregion
```